manager/assets/modext/util/utilities.js

```diff
@@ -319,6 +319,26 @@ Ext.override(Ext.form.Action.Submit,{
     }
 });
 
+Ext.override(Ext.data.Connection, {
+    handleResponse: function(j) {
+        this.transId = false;
+        var k = j.argument.options;
+        j.argument = k ? k.argument : null;
+        // trim out the enclosing textarea tag (the long way to avoid any tags in the response body) if exists
+        j.responseText = j.responseText.replace('<textarea>', '');
```

Comment:
I suspect this would also remove textarea's from within the content (say, someone having a form markup in a chunks' content). 

Is there a different way to handle `j.responseText` here to fix the issue, which doesn't require the `<textarea>` markup to be returned from the processor? Maybe that can be dynamically added in this handleResponse method or elsewhere to fix the issue?

Reply:
Noted. I added this in to restore 'normal behaviour ' after the enclosing above. Back to square one then : s